tests/test_claude_wrapper_hooks.py

```
@@ -575,6 +593,49 @@ def test_live_socket_injects_supported_hooks_without_unlocking_bypass(failures:
+    expect(
+        not settings_arg.startswith("{"),
+        "user-scoped settings: expected --settings to receive a generated file path, got inline JSON",
+        failures,
+    )
+    expect(
+        "/Library/Application Support/cmux/claude/" in settings_arg,
+        f"user-scoped settings: expected Application Support cmux path, got {settings_arg!r}",
+        failures,
+    )
+    expect(
+        not settings_arg.startswith("/tmp/") and not settings_arg.startswith("/private/tmp/"),
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)